### ISSUE TITLE
arrested developmental stage pattern

### ISSUE THREAD

**Issue by pfey03** [obophenotype/upheno#624]
In Dicty, we have mutants with certain developmental arrests, mutants just stop developing beyond a certain point.

For example, the 'mound' is an earlier developmental stage on the way to becoming a fruiting body.
DDPHENO:0000026:development arrests at mound stage

**Pattern**
abnormallyArrestedDevelopmentalStageInLocation

@sbello pointed me to an arrested mammalian phenotype
http://www.ontobee.org/ontology/MP?iri=http://purl.obolibrary.org/obo/MP_0001730

Does this sound useful for others?

Thanks!

**Comment by matentzn** [comment]
I suggest this:
abnormallyArrestedDevelopmentAtStageInLocation:

```
has part some (arrested and (inheres in some (development and during some stage)) and (has modifier some abnormal)) 
```

@dosumis @pfey03 @mah11 @sbello @ybradford @chris-grove (and everyone else). Makes sense?

**Comment by dosumis** [comment]
GO has.

* 'developmental process': A biological process whose specific outcome is the progression of an integrated living unit: an anatomical structure (which may be a subcellular structure, cell, tissue, or organ), or organism over time from an initial condition to a later condition. 

* 'multicellular organism development': The biological process whose specific outcome is the progression of a multicellular organism over time from an initial condition (e.g. a zygote or a young adult) to a later condition (e.g. a multicellular animal or an aged adult).

Using the 'developmental process' would mean  this applies to arrest in development of some specific structure.  

'multicellular organism development' is good for metazoans and plants.  Dicty  should use '[socially cooperative development](http://purl.obolibrary.org/obo/GO_0099120)'

**Comment by mah11** [comment]
I like it. Could something analogous work for cell cycle arrest? FYPO has a lot of cell cycle arrest phenotypes that need better logical defs than they've got at the moment. (I can open a new ticket if this is too much of a tangent here.)

**Comment by dosumis** [comment]
Sounds v.similar.  Could presumably use GO CC process &  phase terms.

**Comment by pfey03** [comment]
@matentzn I like your pattern.

The GO process would be [sorocarp development](http://www.ontobee.org/ontology/GO?iri=http://purl.obolibrary.org/obo/GO_0030587)

**Comment by pfey03** [comment]
Maybe:
abnormalBiologicalProcessAtStageInLocation

Would possibly be more widely applicable?
@mah11 if it could be Stage OR Phase this might fit cell cycle arrests?

**Comment by matentzn** [comment]
sounds reasonable.

**Comment by mah11** [comment]
Agreed; for yeasts we don't need the "InLocation" part.

**Comment by pfey03** [comment]
If Stage can be an anatomy term I also won't need inLocation, at least not for the phenotypes listed above. But maybe we have both:
abnormallyArrestedBiologicalProcessAtStage
abnormallyArrestedBiologicalProcessAtStageInLocation

What different inputs are allowed for Stage and Location?

**Comment by matentzn** [comment]
I suspect the question of stage will need some work. First pass: UBERON [life cycle stage](https://www.ebi.ac.uk/ols/ontologies/uberon/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0000105), but probably we need something that covers cell cycle stages as well; Thats a question for @dosumis 

Location can be absolutely anything historically (any independent continuant). So it could be the entity in the cell in which the process is arrested, or the whole organism. Note that technically speaking the meaning of `in location` is really: the process happens in there (the development IN the cell is arrested); not the development OF that the cell ia arrested. I guess we are a bit hand-wavey about the details here.

**Comment by dosumis** [comment]
> we need something that covers cell cycle stages as well; Thats a question for @dosumis

Use GO cell cycle phase terms.  Should have  a separate pattern for these.

**Comment by matentzn** [comment]
ok so we need:

- [ ] abnormallyArrestedBiologicalProcessAtDevelopmentStage (with Uberon life cycle stage)
- [ ] abnormallyArrestedBiologicalProcessAtDevelopmentStageInLocation (with Uberon life cycle stage)
- [ ] abnormallyArrestedBiologicalProcessDuringCellCyclePhase (with go [cell cycle phase](https://www.ebi.ac.uk/ols/ontologies/go/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0044848&viewMode=All&siblings=false))

Is this what you mean David?

**Comment by dosumis** [comment]
I'd like to see some more examples of locations.  

Mostly i think we're talking about arrest in development of whole animal (or whole colony in case of dicty).  Arrested development of some specific structure while development of the rest of the organism continues is, I think, quite rare.    @pfey03 for your use cases is only some part of colony development blocked (e.g. sorocarp blocked by stalk develops?)?

At the risk of proliferating patterns, maybe better to have some simple patterns for the common, general cases:

abnormallyArrestedDevelopment: GOBP: multicellular organism development  var = Uberon stage
abnormallyArrestedColonialDevelopment: GOBP: multicellular organism development  var = Uberon stage ??? (Does this support dicty stages?)
abnormallyArrestedCellCycle: GOBP: Cell cycle var  =  [cell cycle phase](https://www.ebi.ac.uk/ols/ontologies/go/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0044848&viewMode=All&siblings=false)

Happy to add more specific patterns too - but perhaps collect some examples from major MODs first?

**Comment by pfey03** [comment]
@dosumis 
Uberon does not support Dicty stages. We use the Dicty anatomy for all these. @matentzn  asked if those could be aligned (https://github.com/obophenotype/dicty-phenotype-ontology/issues), but not sure. As you know Dicty anatomy quite specific for Dictyostelids and some overlap to fungi.

**Comment by pfey03** [comment]
@dosumis as you know, Dicty has a developmental cycle for survival of adverse conditions. But since it's a cycle that ends with spores that start the new cycle, there are mutants that stop development at specific developmental stages. It is hard to express the exact phenotypes in other ways. 
Arrests at mound, loose mound, tight mound and tipped mound as well as slug stage are currently annotated to 142 mutants. So these arrests in development are quite common especially when signaling networks or important structural components are interrupted.

**Comment by pfey03** [comment]
@dosumis 
Maybe I can use this pattern for, e.g. 'development arrests at tipped mound stage'
- abnormallyArrestedBiologicalProcessInLocation
If I can use DDANAT as Location
**abnormallyArrested GO:0030587 sorocarp development In DDANAT:0000006 tipped mound**

- AtLocation would fit me better than InLocation

Or a new pattern:
- abnormallyArrestedBiologicalProcessAtAnatomicalEntity

Thanks, I'd like to move this along

**Comment by sbello** [comment]
Discussed during the Upheno call on 12/17/20 we would like this to be created

**Comment by dosumis** [comment]
General patterns:
- [ ] has part some (arrested and (inheres in some (development and during some stage)) and (has modifier some abnormal)) 
- [ ] has part some (rate and (inheres in some (development and during some stage)) and (has modifier some abnormal)) 

For whole organism arrested development 
 - [ ]  has part some (arrested and (inheres in some (multicellular organism development' and during some stage)) and (has modifier some abnormal)) 

For whole organism abnormal developmental rate:
 - [ ] has part some (rate and (inheres in some ('multicellular organism development' and during some stage)) and (has modifier some abnormal)) 

For dicty:
 - has part some (arrested and (inheres in some ('socially cooperative development' and during some stage)) and (has modifier some abnormal))   # But for this to work, strictly we need stage terms for dicty, whereas right now dicty has anatomy terms for this.
 We could fix by adding a set of stage terms in a distinct ontology (e.g. FBdv, Worm development ontology) or use:
 - has part some (arrested and (inheres in some ('socially cooperative development' and occurs_in some <anatomy stage term>)) and (has modifier some abnormal))

Petra to decide which approach to use.

**Comment by pfey03** [comment]
I'll create a stage ontology, choosing the owl compatible first version.

**Comment by nicolevasilevsky** [comment]
@dosumis- I created these two general patterns:
https://github.com/obophenotype/upheno/pull/737
https://github.com/obophenotype/upheno/pull/736

Do we need the two 'whole organism arrested development' patterns too? GO:0007275 multicellular organism development
 is a child of GO:0032502 developmental process, so the patterns above could be use for 'whole organism arrested development' and 'whole organism abnormal developmental rate'.

**Comment by pfey03** [comment]
@nicolevasilevsky, for dicty, sorocarp development is not under multicellular organism development. Maybe this is needed for more complex organisms. 
For us 'abnormalRateOfDeveopmentDuringStage' is perfect.
Can I start using this with stage ontology now?
Thanks!

**Comment by nicolevasilevsky** [comment]
Hi @pfey03 I merged the two PRs, so yes, I believe you can start using those patterns now

**Comment by pfey03** [comment]
Thanks @nicolevasilevsky
